alfresco-transform-tika/alfresco-transform-tika/src/main/java/org/alfresco/transformer/tika/parsers/ExifToolParser.java

```diff
@@ -0,0 +1,260 @@
+package org.alfresco.transformer.tika.parsers;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.Reader;
+import java.net.URL;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.tika.metadata.Metadata;
+
+import org.apache.tika.exception.TikaException;
+import org.apache.tika.io.IOUtils;
+import org.apache.tika.io.NullOutputStream;
+import org.apache.tika.io.TemporaryResources;
+import org.apache.tika.io.TikaInputStream;
+import org.apache.tika.parser.ParseContext;
+import org.apache.tika.parser.external.ExternalParser;
+import org.apache.tika.parser.external.ExternalParsersFactory;
+import org.apache.tika.sax.XHTMLContentHandler;
+import org.xml.sax.ContentHandler;
+import org.xml.sax.SAXException;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class ExifToolParser extends ExternalParser {
+    
+    private static final String EXIFTOOL_PARSER_CONFIG = "parsers/external/config/exiftool-parser.xml";
+
+    public ExifToolParser() throws IOException, TikaException {
+        super();
+        ExternalParser eParser = ExternalParsersFactory.create(getExternalParserConfigURL()).get(0);
+        this.setCommand(eParser.getCommand());
+        this.setIgnoredLineConsumer(eParser.getIgnoredLineConsumer());
+        this.setMetadataExtractionPatterns(eParser.getMetadataExtractionPatterns());
+        this.setSupportedTypes(eParser.getSupportedTypes());
+    }
+    
+    private URL getExternalParserConfigURL(){
+        ClassLoader classLoader = ExifToolParser.class.getClassLoader();
+        return classLoader.getResource(EXIFTOOL_PARSER_CONFIG);
+    }
+
+    /**
+     * Adapted from {@link org.apache.tika.parser.external.ExternalParser} 

```

Comment:
Are we in a position (enough knowledge) to also raise at least a speculative issue within Tika project ? Initially speculative, since to fully understand & resolve, someone in the Tika team may ask for a standalone unit test case that demonstrates the potential bug.

I think @ancutam has also done something similar previously.

This may allow us to remove this adaptation in the future (if resolved in a future release of Tika).